src/Type/Php/RegexArrayShapeMatcher.php

```diff
@@ -31,6 +31,10 @@
  */
 final class RegexArrayShapeMatcher
 {
+	/**
+	 * Pass this into $flagsType as well if the library supports emulating PREG_UNMATCHED_AS_NULL on PHP 7.2 and 7.3
+	 */
+	public const PREG_UNMATCHED_AS_NULL_ON_72_73 = 2048;
```

Comment:
The highest in use currently is 256, so I chose 2048 to leave 512/1024 as gaps if PHP should add new constants there.